docs/BIDS.md

```diff
@@ -40,3 +44,22 @@ BIDS_Dataset/
 ├── ...
 └── participants.tsv
 ```
+
+## Validation of BIDS datasets
+
+[bids-validator](https://github.com/bids-standard/bids-validator) can be run to ensure that a dataset is BIDS-compliant.
+
+Some errors and files can be ignored by the validator in respectively `.bids-validator-config.json` and 
+`.bidsignore` files at the root of the BIDS folder.
+These files are automatically generated by clinica converters to ignore the following issues:
+- Won't fix errors:
```

Comment:
It needs some background to be understood:

```suggestion
Clinica provides tools to curate several publicly available neuroimaging datasets ([ADNI](../Converters/ADNI2BIDS/), [AIBL](../Converters/AIBL2BIDS/), [NIFD](../Converters/NIFD2BIDS/) and [OASIS](../Converters/OASIS2BIDS/)) and convert them to BIDS. We decided to ignore several warnings and errors detected by the validator. These are listed in the `.bids-validator-config.json` and `.bidsignore` files at the root of each BIDS folder. These files are automatically generated by Clinica converters to ignore the following issues:

- Won't fix errors:
```

(New line missing before list)